manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -244,6 +244,21 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  // The following two tests do not produce documentation, but had no better module to live in. When we have an
+  // enterprise-cypher module, let's move these tests
```

Comment:
Cypher tests for enterprise only features are in the enterprise kernel module. I think that is a better place for these tests.